vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -434,8 +435,10 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
         for (auto* ex : data.actions) { ex->interactions = &data.generated_interactions; }
       }
 
+      // shared, action, action, slot
       data.include_list.clear();
-      build_cb_example<is_learn>(data.cb_ex, slot, data);
+      build_cb_example<is_learn>(
+          data.cb_ex, slot, data.stored_labels[1 /* shared */ + data.actions.size() + slot_id], data);
```

Comment:
can the label's index go out of bounds somehow?

Reply:
It should not as that vector is produced based on those other values.

Reply:
Ill add an assert